native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/Info.plist

```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<string>go.meethour.io.ios.sdk</string>
```

Comment:
@chepury-bharathraj  - Let this pick from $(PRODUCT_BUNDLE_IDENTIFIER)

Reply:
Ok